### PR TITLE
adjusted config load to check for test prefix

### PR DESCRIPTION
**Pull request by vigan-abd**
This PR adds support for `test.<configname>.json` files that would be loaded in case if `env` is `test`

### CODE REVIEW

File: base.js [hunk 1]
```diff
@@ -49,10 +50,16 @@ class Base extends EventEmitter {
...
-    _.merge(
-      this.conf,
-      this.getConf(this.ctx.env, group, `${this.ctx.root}/config/${c}.json`)
-    )
+    const fprefix = this.ctx.env === 'test' ? 'test' : ''
+    const dirname = join(this.ctx.root, 'config')
+
+    let confpath = join(dirname, `${c}.json`)
+    const testpath = join(dirname, `${fprefix}.${c}.json`)
+    if (fprefix && fs.existsSync(testpath)) {
```
Comment (robertkowalski) [inline base.js:58]: hmmm... this would lead to a silent error when a testfile is not there, but the user starts the service with the --env=test arg

**Review by robertkowalski — APPROVED** [review]
lgtm

File: base.js (hunk 1 above)
Comment (robertkowalski) [reply base.js:58]: on the other side, this is intented for backwards compat and where a seperate test* conf file is not requried

Comment (vigan-abd) [reply base.js:58]: yep the idea was to keep backward compatibility also so nothing breaks if test file doesn't exist

**Review by ezewer — APPROVED** [review]
looks good to me!

**Review by batrudinych — APPROVED** [review]

**Pull request merged**